custom_reports/columns.rst

```diff
@@ -47,3 +47,62 @@ dimensions to metrics or different dimensions.
 
 .. include:: columns__dimension_to_metric_transformations.rst
 .. include:: columns__dimension_to_dimension_transformations.rst
+
+Exporter
+--------
+
+The tables in the following sections shows all columns that may be easily used in Exporter config.
+For sessions and events scope there is preferred list of columns due to performance and reliability reasons.
+
+.. note::
+    The table below presents a short description of the selected fields.
+
++--------------------------------+-------------------------------------------------------+
+| Column                         | Description                                           |
++================================+=======================================================+
+| app_uuid                       | Binary format (16 bytes).                             |
++--------------------------------+-------------------------------------------------------+
+| server_time                    | In UTC.                                               |
++--------------------------------+-------------------------------------------------------+
+| location_ipv4                  | May be partially anonymized. Binary format (4 bytes). |
++--------------------------------+-------------------------------------------------------+
+| location_ipv6                  | May be partially anonymized. Binary format (16 bytes).|
++--------------------------------+-------------------------------------------------------+
+| session_custom_dimension_1     | Number of custom dimension slots is configurable.     |
++--------------------------------+-------------------------------------------------------+
+| event_custom_dimension_1       | Number of custom dimension slots is configurable.     |
++--------------------------------+-------------------------------------------------------+
+| session_custom_variable_key_1  | Number of custom variable slots is configurable.      |
++--------------------------------+-------------------------------------------------------+
+| event_custom_variable_key_1    | Number of custom variable slots is configurable.      |
++--------------------------------+-------------------------------------------------------+
+| event_index                    | Starts from 0.                                        |
++--------------------------------+-------------------------------------------------------+
+| page_view_index                | Starts from 0.                                        |
++--------------------------------+-------------------------------------------------------+
+
+
+Sessions scope - preferred
+``````````````````````````
+
+.. include:: columns__exporter_preferred_sessions_columns.rst
```

Comment:
would it be possible to include plain `json` file? it would be easier to generate and more universal. but if not, rst is ok :+1:

Reply:
It will change nothing as still `.. code-block:: json` will be needed

Reply:
but cannot it be something like
```suggestion
.. code-block:: json
    "columns": .. include:: columns__exporter_preferred_sessions_columns.json
```
? most probably can't, but worth asking :smile:

Reply:
Nope, `..include` needs to be first, generally it can't be nested this way